pkg/yang/types.go

```diff
@@ -139,6 +139,16 @@ func (t *Typedef) resolve() []error {
 		y.Default = t.Default.Name
 	}
 
+	if t.Type.IdentityBase != nil {
+		// We need to copy over the IdentityBase statement if the type has one
+		if idBase, err := RootNode(t).findIdentityBase(t.Type.IdentityBase.Name); err == nil {
+			y.IdentityBase = idBase.Identity
+		} else {
+			return []error{fmt.Errorf("Could not resolve identity base for typedef: %s",
+				t.Type.IdentityBase.Name)}
```

Comment:
Ah, interesting - I have had the 80-char-maximum drilled into me clearly :-) Removed this linebreak.